packages/rolldown/tests/cli/__snapshots__/cli-e2e.test.ts.snap

```diff
@@ -7,46 +7,46 @@ USAGE rolldown -c <config> or rolldown <input> <options>
 
 OPTIONS
 
-  --config -c, <filename>     Path to the config file (default: \`rolldown.config.js\`).
```

Comment:
These indentation should be preserved for better display.

Reply:
Done~